aries_cloudagent/connections/models/diddoc/schemas/verificationmethodschema.py

```diff
@@ -0,0 +1,15 @@
+from marshmallow import Schema, fields
+
+
+class VerificationMethodSchema(Schema):
+    """
+
+    Example
+    {"type": "Ed25519VerificationKey2018",
+     "id": "did:sov:CYQLsccvwhMTowprMjGjQ6#key-1",
+     "publicKeyBase58": "CLFRfp2wa3ifbsVvdq52WcpEy7aujactsoqQgxkz7ZKR"}
+    """
+    id = fields.Str()
+    type = fields.Str()
+    publicKeyBase58 = fields.Str()
```

Comment:
Verification methods will have other key types and key representations such as publicKeyPem, publicKeyHex, etc.

Reply:
Yes, I observed that it is pretty similar to publicKey schema. I think that the publicKey schema could be the same for both. What do you think?

Reply:
Yeah, they are essentially the same.. Perhaps an alias or a subclass would be good so we still have a separate `VerificationMethodSchema` but just reusing `PublicKey`